tf-modules/ingest/reporter-functions.tf

```diff
@@ -0,0 +1,32 @@
+module "report_executions" {
```

Comment:
Ideally these modules would be part of the archive module, because they are really only thin lambda wrappers around DynamoDB PUT operations, but I did not have the time to resolve the dependencies that would bring today.

Reply:
Done.